tests/exporters/tflite/test_exporters_tflite_cli.py

```diff
@@ -83,20 +88,73 @@ class TFLiteCLIExportTestCase(unittest.TestCase):
+        quantization: Optional[str] = None,
+        fallback_to_float: bool = False,
+        inputs_dtype: Optional[str] = None,
+        outputs_dtype: Optional[str] = None,
+        calibration_dataset_name_or_path: Optional[str] = None,
+        calibration_dataset_config_name: Optional[str] = None,
+        num_calibration_samples: int = 200,
+        calibration_split: Optional[str] = None,
+        primary_key: Optional[str] = None,
+        secondary_key: Optional[str] = None,
+        question_key: Optional[str] = None,
+        context_key: Optional[str] = None,
+        image_key: Optional[str] = None,
+    ):
         with TemporaryDirectory() as tmpdir:
+            command = f"python3 -m optimum.exporters.tflite --model {model_name}".split()
             if task is not None:
-                subprocess.run(
-                    f"python3 -m optimum.exporters.tflite --model {model_name} --task {task} {shapes} {tmpdir}",
-                    shell=True,
-                    check=True,
-                )
+                command.append(f"--task={task}")
+            if quantization is not None:
+                command.append(f"--quantize={quantization}")
+            if fallback_to_float:
+                command.append("--fallback_to_float")
+            if inputs_dtype is not None:
+                command.append(f"--inputs_type={inputs_dtype}")
+            if outputs_dtype is not None:
+                command.append(f"--outputs_type={outputs_dtype}")
+            if calibration_dataset_name_or_path is not None:
+                command.append(f"--calibration_dataset={calibration_dataset_name_or_path}")
+            if calibration_dataset_config_name is not None:
+                command.append(f"--calibration_dataset_config_name={calibration_dataset_config_name}")
+            if calibration_split is not None:
+                command.append(f"--calibration_split={calibration_split}")
+            if primary_key is not None:
+                command.append(f"--primary_key={primary_key}")
+            if secondary_key is not None:
+                command.append(f"--secondary_key={secondary_key}")
+            if question_key is not None:
+                command.append(f"--question_key={question_key}")
+            if context_key is not None:
+                command.append(f"--context_key={context_key}")
+            if image_key is not None:
+                command.append(f"--image_key={image_key}")
+            command.append(f"--num_calibration_samples={num_calibration_samples}")
+
+            tflite_config = tflite_config_constructor(AutoConfig.from_pretrained(model_name))
+            mandatory_axes = tflite_config.get_mandatory_axes_for_task(task)
+            shapes = [f"--{name}={getattr(tflite_config, name)}" for name in mandatory_axes]
+            command += shapes
+            command.append(tmpdir)
+
+            proc = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
No need to do it now, but in a following PR we could remove subprocess calls as well.